index.html

```diff
@@ -71,7 +71,7 @@
       <a id="statement-link" href="https://security.intuit.com/privacy" target="_blank">Privacy Statement</a>
       </span>
     </div>
-
+  </main>
```

Comment:
Well done implemented the requested changes :+1: Read more on[ semantic html elements here.](https://internetingishard.com/html-and-css/semantic-html/)